kong/constants.lua

```diff
@@ -282,6 +282,15 @@ local constants = {
   },
 
   SOCKET_DIRECTORY = "sockets",
+  SOCKETS = {
+    WORKER_EVENTS = "we",
```

Comment:
Should we add a short ext name for these files? These name may be hard to recognize  when debugging. like `we.s`

Reply:
By default now they live inside a subdirectory called "sockets" so we'll know based on file path. Initially i had a file extension but removed to keep these paths as short as possible. @flrgh what do you think? @chronolaw is the directory name enough?

Reply:
> they live inside a subdirectory called "sockets" so we'll know based on file path

:+1: this was exactly my thought